src/App.tsx

```diff
@@ -10,33 +11,22 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
-
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
+  const [selectedTabId, setSelectedTabId] = useState(1);
```

Comment:
the first tab is selected by default
```suggestion
  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
```